homeassistant/components/ecobee/climate.py

```diff
@@ -253,6 +262,24 @@ def fan_min_on_time_set_service(service: ServiceCall) -> None:
             thermostat.set_fan_min_on_time(str(fan_min_on_time))
 
             thermostat.schedule_update_ha_state(True)
+    
+    def aux_cutover_threshold_set_service(service: ServiceCall) -> None:
+        """Set temperature at which the system will switch to auxilary heating."""
+        entity_id = service.data.get(ATTR_ENTITY_ID)
+        
+        aux_cutover_threshold = service.data[ATTR_AUX_CUTOVER_THRESHOLD]
```

Comment:
Looks like `aux_cutover_threshold` is no longer used here and can be removed.

Reply:
Thanks.